src/Migrations/Version20191122163710.php

```diff
@@ -22,14 +22,14 @@ public function up(Schema $schema) : void
         // this up() migration is auto-generated, please modify it to your needs
         $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
 
-        $this->addSql('CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_8D93D649E7927C74 (email), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB');
+        $this->addSql('ALTER TABLE user ADD linkedin VARCHAR(255) DEFAULT NULL, CHANGE roles roles JSON NOT NULL, CHANGE homepage homepage VARCHAR(255) DEFAULT NULL, CHANGE password_changed password_changed DATETIME DEFAULT NULL');
```

Comment:
Duomenų bazių migracijų esmė – pridėti naujus failus, o ne viską seną ištrinti ir pakeisti nauju.

Dėl to testuose (reliame projekte analogiškai būtų pas kolegą) migracija nepavyksta:
![image](https://user-images.githubusercontent.com/1453957/70868357-bb809180-1f88-11ea-94d4-134b4379f5df.png)

Todėl ir negaliu iki galo įvertinti viso darbo

Reply:
Plačiau:
 * https://symfony.com/doc/master/bundles/DoctrineMigrationsBundle/index.html